### PR TITLE
fix: remove `@internationalized/date` from list of peer dependencies and make it just dependency

### PR DESCRIPTION
**Pull request by MellKam**
resolves: #797

### CODE REVIEW

File: packages/radix-vue/package.json [hunk 1]
```diff
@@ -57,17 +57,12 @@
...
   },
-  "peerDependenciesMeta": {
-    "@internationalized/date": {
-      "optional": true
-    }
-  },
   "dependencies": {
     "@floating-ui/dom": "^1.5.4",
     "@floating-ui/vue": "^1.0.4",
+    "@internationalized/date": "^3.5.2",
```
Comment (zernonia) [inline packages/radix-vue/package.json:65]: I saw @epr3 updated to use `"@internationalized/date": "^3.5.1"` in devDependencies, perhaps we should use the same here?

Comment (MellKam) [reply packages/radix-vue/package.json:65]: Do you want to change it to 3.5.1? We don't need it to be dev dependency now. What do you suggest? 3.5.2 is the latest version, should be fine

Comment (epr3) [reply packages/radix-vue/package.json:65]: Yes, use the latest version.  👍

Comment (zernonia) [reply packages/radix-vue/package.json:65]: Cool! Just wanna make sure the version is the same as in the dev version.

**Pull request merged**